tests/naive.py

```diff
@@ -559,17 +597,27 @@ def multi_distance_profile(query_idx, T, m, include=None, discords=False):
     return D_prime_prime
 
 
-def mstump(T, m, excl_zone, include=None, discords=False):
+def mstump(T, m, excl_zone, include=None, discords=False, T_subseq_isconstant=None):
     T = T.copy()
 
     d, n = T.shape
     k = n - m + 1
 
+    if T_subseq_isconstant is None or callable(T_subseq_isconstant):
+        T_subseq_isconstant = [T_subseq_isconstant] * d
+    # else means T_subseq_isconstant is list or a numpy 2D array
```

Comment:
@seanlaw 
This is naive function. So, I assume you are okay with having the comment above in the code. If not, please let me know.